dcs-packaging-tool/dcs-packaging-tool-gui/src/main/java/org/dataconservancy/packaging/gui/view/impl/CreateNewPackageViewImpl.java

```diff
@@ -65,7 +61,8 @@ public CreateNewPackageViewImpl(Labels labels) {
         super(labels); 
         this.labels = labels;
         
-        getContinueButton().setText(labels.get(LabelKey.CONTINUE_BUTTON));
+        getContinueButton().setText(labels.get(LabelKey.NEXT_BUTTON));
```

Comment:
You're just changing the constant's name right? Then Don't worry about. Sounds reasonable. I thought there might be other reason involving the use of both NEXT and CONTINUE.